src/topology/instances/ennreal.lean

```diff
@@ -409,7 +409,7 @@ lemma finset_sum_supr_nat {α} {ι} [semilattice_sup ι] {s : finset α} {f : α
 begin
   refine finset.induction_on s _ _,
   { simp,
-    exact (bot_unique $ supr_le $ assume i, le_refl ⊥).symm },
+    exact (@bot_unique ℝ≥0∞ _ _ $ supr_le $ assume i, le_refl ⊥).symm },
```

Comment:
This is a tangential newbie question, but I ran into a similar thing in #6775 where inserting a class in the hierarchy caused a few lemmas not to elaborate without an extra hint like this. Assuming what I ran into is related, is this sort of thing expected?

Reply:
I don't know what's going on here. :-(

Reply:
Could it be caused by these lemmas now taking a stronger assumption then they need to?

Reply:
Seems it's just not necessary.